python/katana/__init__.py

```diff
@@ -23,7 +23,9 @@
 
 # Initialize the galois runtime immediately.
 import katana.galois
+from katana.galois import getVersion
 
+__version__ = getVersion()
```

Comment:
No need for this import. Just use the full module path since it's only referenced once.